addons/mass_mailing/views/mailing_templates_portal_unsubscribe.xml

```diff
@@ -61,7 +61,7 @@
                                     <li t-foreach="(lists_optin + lists_optout)"
                                         t-as="mailing_list"
                                         class="list-group-item d-flex align-items-center">
-                                        <t t-set="title" t-value="mailing_list.name if mailing_list.is_public else 'Mailing List #%s' % mailing_list.id"/>
+                                        <t t-set="title" t-value="mailing_list.name"/>
```

Comment:
I don't know about this because maybe you subscribed to a mailing list a while ago, and it was unpublished and renamed "People who I can sell this thing for twice the price" in the back-end. As "unpublished" implies to the user nobody will see how it's named it's better to never display that.